### ISSUE TITLE
Hide Images is broken since last update

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#5270]
Logged via : https://bugzilla.mozilla.org/show_bug.cgi?id=1405913#c2

Firefox ios v 18.1 (15719) Hide Images is broken since last update

Since the ios update on the 17th July 2019, the Hide Images option does not work. Firefox ios v 18.1 (15719) 

Steps to reproduce:

    1.  Start FF
    2. Settings -> enable Hide Images
    3. Open any website.
    4. All images are loaded and displayed
    5. Settings -> Toggle Hide Images : Result Images are hidden.
    6. Open new tab with Hide Images enabled -> Images are loaded and displayed. 

In other words, I have to load a site with images first before I can hide the images, which defeats the purpose. Before the update I could open sites with Hide Images enabled and the images did not display ( and I hope were not downloaded).

 and I've stopped surfing, because I only used Firefox because I have a 500Mb mobile data plan per month.

**Comment by supersophie** [comment]
Hi,  This is closed.

when will this be released to ios devices?

**Comment by SimonBasca** [comment]
Verifying as fix on 18.2 (158817)

**Comment by SimonBasca** [comment]
> Hi, This is closed.
> 
> when will this be released to ios devices?

This will be released in the next version.
